src/validation.cpp

```diff
@@ -4386,7 +4395,8 @@ bool LoadMempool(void)
             if (nTime + nExpiryTimeout > nNow) {
                 LOCK(cs_main);
                 AcceptToMemoryPoolWithTime(chainparams, mempool, state, tx, nullptr /* pfMissingInputs */, nTime,
-                                           nullptr /* plTxnReplaced */, false /* bypass_limits */, 0 /* nAbsurdFee */);
+                                           nullptr /* plTxnReplaced */, false /* bypass_limits */, 0 /* nAbsurdFee */,
+                                           nullptr /* test_accept */);
```

Comment:
Already default value. Or remove default value from declaration?

Reply:
Note this is ATMPWT, not ATMP ;)

Reply:
Ah!